Source/controls/touch.cpp

```diff
@@ -250,7 +250,7 @@ static void preprocess_finger_up(SDL_Event *event)
 			// but only if the finger hasn't moved since it was pressed down by more than MAX_TAP_MOTION_DISTANCE pixels
 			float xrel = ((event->tfinger.x * devilution::GetOutputSurface()->w) - (finger[port][i].last_down_x * devilution::GetOutputSurface()->w));
 			float yrel = ((event->tfinger.y * devilution::GetOutputSurface()->h) - (finger[port][i].last_down_y * devilution::GetOutputSurface()->h));
-			float max_r_squared = (float)(MAX_TAP_MOTION_DISTANCE * MAX_TAP_MOTION_DISTANCE);
+			auto max_r_squared = (float)(MAX_TAP_MOTION_DISTANCE * MAX_TAP_MOTION_DISTANCE);

```

Comment:
static_cast<float> instead of c-style cast.

I know it doesn't make a difference for numeric types, but it makes looking for casts easier. Plus at some point maybe someone writes a numeric_cast that checks for truncation and similar issues, then it makes this easier to find.